json-form-react/JsonForm.js

```diff
@@ -53,22 +53,46 @@ const JsonForm = (props) => {
     const fieldClasses = `jsfr-field ${field.class} ${formErrors[field.name] ? 'jsfr-field-error' : ''}`;
     const fieldWrapperClasses = `jsfr-field-wrapper ${field.wrapperClass} ${formErrors[field.name] ? 'jsfr-field-wrapper-error' : ''}`;
 
-    return (
-      <div key={field.name} className={fieldWrapperClasses}>
-        <label htmlFor={fieldId}>
-          {field.label}
-        </label>
-
-        <input
-          id={fieldId}
-          type={field.type}
-          name={field.name}
-          value={formData[field.name] || ''}
-          placeholder={field.placeholder}
-          className={fieldClasses}
-          onChange={onChange}/>
-      </div>
-    );
+    if (field.type == 'radio') {
```

Comment:
@yasikahivin Let's use `===` since `field.type should always be a string.

Reply:
noted @cie6868